compiler/src/iree/compiler/Dialect/Flow/Transforms/Passes.td

```diff
@@ -312,4 +306,14 @@ def VerifyInputLegality: Pass<"iree-verify-input-legality", ""> {
   let constructor = "mlir::iree_compiler::IREE::Flow::createVerifyInputLegalityPass()";
 }
 
+def FuseDequantizationMatmul:
```

Comment:
keep these alphabetized (where it was) - but ideally these should go in the global optimize pipeline